src/mcts/search.cc

```diff
@@ -542,17 +541,17 @@ void Search::WatchdogThread() {
       // mode during thinking.
       // Minimum wait time is there to prevent busy wait and other thread
       // starvation.
-      watchdog_cv_.wait_for(lock.get_raw(), remaining_time,
-                            [this]()
-                                NO_THREAD_SAFETY_ANALYSIS { return stop_; });
+      watchdog_cv_.wait_for(lock.get_raw(), remaining_time, [this]() {
+        return stop_.load(std::memory_order_acquire);
+      });
     }
     MaybeTriggerStop();
   }
   MaybeTriggerStop();
 }
 
 void Search::FireStopInternal() REQUIRES(counters_mutex_) {
```

Comment:
Doesn't really require `counters_mutex_` anymore, so this annotation can be removed.

Reply:
Done